runtime/lua/vim/lsp/handlers.lua

```diff
@@ -149,6 +149,28 @@ M['workspace/applyEdit'] = function(_, _, workspace_edit)
   }
 end
 
+--@see https://microsoft.github.io/language-server-protocol/specifications/specification-current/#workspace_configuration
+M['workspace/configuration'] = function(err, _, params, client_id)
+  local client = vim.lsp.get_client_by_id(client_id)
+  if err then error(tostring(err)) end
```

Comment:
```suggestion
  if err then error(vim.inspect(err)) end
```

Afaik `error` is a table with the code and message, so `tostring(err))` would return `table: 0x...`.